api/storage/file.go

```diff
@@ -101,6 +104,10 @@ func (p PresignUrlOptions) isValid() error {
 		return fmt.Errorf("invalid mode: %d", p.Mode)
 	}
 
+	if p.Expiry > 604800 {
+		return fmt.Errorf("invalid expiry, cannot exceed 7 days: %d", p.Expiry)
+	}
```

Comment:
Good catch on the comments from the node-sdk on the max expiry. For consistency we'd normally handle this kind of validation on the nitric server side which saves re-implementing this across all our SDKs, this also allows these limits to be changed when possible in one location.

```suggestion
```

Reply:
Yeah, that makes sense. It didn't cross my mind as I was referring to the node-sdk.

Reply:
Just been running some tests, looks like everything is working well. If you remove that validation code I think this should be good to merge :)

Reply:
That would be great! I have made the requested changes, please go through them

Reply:
awesome! thanks for that, going through it now.